frontend/src/metabase/dashboard/selectors/selectors.js

```diff
@@ -17,7 +17,11 @@ import { getEmbedOptions, getIsEmbedded } from "metabase/selectors/embed";
 import Question from "metabase-lib/Question";
 
 import { isVirtualDashCard } from "../utils";
-import { getDashboardId } from "./selectors-typed";
+import {
```

Comment:
This is technically a circular dependency, but jest seemed to run fine for me locally.

@uladzimirdev I think I saw you working on circular dependencies, do you have any idea if this is a problem?

Reply:
I suggest you take a look if your test would fail on the CI. I've encountered the same exact problem before, where Cypress runs fine locally, but would fail inevitably on CI.

I don't think circular dependency imports should be left in the code.

Reply:
🤔 not sure if something changed when I updated the branches, but I don't see the circular dependency anymore 🤔 

`selectors.js` imports from `selectors-typed` which is fine, and `selectors-typed`  doesn't import from the untyped file, and neither should the files imported by it

Reply:
I'd say as long as all required checks on this PR is green, then it'll be safe.